vignettes/admiralpeds.Rmd

```diff
@@ -1,21 +0,0 @@
----
```

Comment:
Instead of deleting this one (as we are going to need a Get Started guide) could you just copy over anything relevant from https://github.com/pharmaverse/admiralonco/blob/main/vignettes/admiralonco.Rmd as a starter please here? we'll need to update later once we know exactly which vignettes we'll have and any other pediatrics specifics

Reply:
Added one back